plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.xml

```diff
@@ -194,6 +194,9 @@
   		class="com.google.cloud.tools.eclipse.appengine.validation.ApplicationQuickAssistProcessor">
   	</quickFixProcessor>
   	<quickFixProcessor
+  	    class="com.google.cloud.tools.eclipse.appengine.validation.VersionQuickAssistProcessor">
```

Comment:
If these `XxxQuickAssistProcessors` differ only by the element name, perhaps there should be a single class (`XsltQuickAssistProcessor`) that uses the `IExecutableExtension` interface to pull out configuration information from this plugin.xml element.  You'd define the instance as:
```
<quickFixProcessor
    class="com.google.cloud.tools.eclipse.appengine.validation.XsltQuickAssistProcessor:version">
```
Your processor would implement `IExecutableExtension#setInitializationData(IConfigurationElement element,Object data)` and use the data (always a String) to pull out the `version` string.